sql/src/main/java/io/crate/operation/reference/information/InformationSchemaExpressionFactories.java

```diff
@@ -235,6 +257,22 @@ public Boolean value() {
                         return null;
                     }
                 })
+            .put(InformationTablesTableInfo.Columns.SELF_REFERENCING_COLUMN_NAME, () -> new
+                RowContextCollectorExpression<TableInfo, BytesRef>() {
+
+                @Override
+                public BytesRef value() {
+                    return null;
```

Comment:
does it make sense to create a `RowCollectNullExpressionFactory` class that is reused?